src/NuGet.Clients/NuGetConsole.Host.PowerShell/PowerShellHost.cs

```diff
@@ -74,24 +74,14 @@ internal abstract class PowerShellHost : IHost, IPathExpansion, IDisposable
         private string _currentSolutionDirectory;
 
         /// <summary>
-        /// An initial restore event used to compare against future (real) restore events. This value endures on
-        /// <see cref="_latestRestore"/> and <see cref="_currentRestore"/> as long as no restore occurs. Note that the
-        /// hash mentioned here cannot collide with a real hash.
-        /// </summary>
-        private static readonly SolutionRestoredEventArgs InitialRestore = new SolutionRestoredEventArgs(
-            isSuccess: true,
-            solutionSpecHash: "initial");
-
-        /// <summary>
         /// This field tracks information about the latest restore.
         /// </summary>
-        private SolutionRestoredEventArgs _latestRestore = InitialRestore;
+        private SolutionRestoredEventArgs _latestRestore;
```

Comment:
I'd prefer not to file an issue for this as I believe it will never be addressed separately without having strong justification. I'd rather leave it as it is until next product change in the same area.